plugins/inputs/couchdb/README.md

```diff
@@ -62,194 +62,231 @@ httpd statistics:
 
 ### Example output:
 
+**Post Couchdb 2.0**
 ```
-➜  telegraf git:(master) ✗ ./telegraf --config ./config.conf  --input-filter couchdb --test
-* Plugin: couchdb,
- Collection 1
-> couchdb,server=http://localhost:5984/_stats couchdb_auth_cache_hits_current=0,
-couchdb_auth_cache_hits_max=0,
```

Comment:
Little bit of README cleanup I have been doing:
- Remove the lines with the command and unrelated bits of output
- Remove the `> ` prefix
- Join all the lines into one big line